src/components/HeroBanner/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { Hyperlink } from '@edx/paragon';
+import './HeroBanner.scss';
+
+
+const ViewOrPurchaseButton = ({
+  authorized,
+  journalAboutId,
+  purchaseUrl,
+  price,
+}) => (
+  authorized ? (
+    <Link to={{ pathname: `/${journalAboutId}` }} className="btn btn-success">View Now</Link>
+  ) : (
+    <Hyperlink destination={purchaseUrl} className="btn btn-success" content={`Purchase Access ($${price})`} />
+  )
+);
+
+const HeroBanner = props => (
+  <div className="hero" style={{ background: `linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ), url(${props.bannerUrl})` }} >
+    <div><h1>{props.title}</h1></div>
+    {
+      props.description &&
+        <div><h2>{props.description}</h2></div>
+    }
+    {
+      props.showButton &&
+        <div>
+          <ViewOrPurchaseButton
+            authorized={props.authorized}
+            journalAboutId={props.journalAboutId}
+            purchaseUrl={props.purchaseUrl}
+            price={props.price}
+          />
+        </div>
+    }
+  </div>
+);
+
+ViewOrPurchaseButton.defaultProps = {
+  authorized: false,
+  journalAboutId: 0,
+  purchaseUrl: '',
+  price: '',
+};
+
+ViewOrPurchaseButton.propTypes = {
+  authorized: PropTypes.bool,
+  journalAboutId: PropTypes.number,
+  purchaseUrl: PropTypes.string,
+  price: PropTypes.string,
+};
+
+
+HeroBanner.defaultProps = {
+  title: '',
+  description: '',
+  bannerUrl: '',
+  showButton: null,
```

Comment:
change this to `showButton: false` and you can just use the `showButton` prop with no value.